terraform.sh

```diff
@@ -26,9 +27,29 @@ if (which pass >/dev/null 2>&1); then
     declare "${secret}"="${pass_secret}"
 fi
 
+if [ -n "${VAULT_ADDR}" ] && [ -z "${VAULT_TOKEN}" ]; then
```

Comment:
Could you explode the `if` in two parts?
```
if [ -n "${VAULT_ADDR}" ]; then
  if [ -z "${VAULT_TOKEN}" ]; then
    ...
  fi
  ...
fi
```

Reply:
In order to include all the logic you added below in the `if [ -n "${VAULT_ADDR}" ] && [ -n "${VAULT_TOKEN}" ]; then` branch *here*.

And declare the key and secrets (in the same way it's done with the `pass` password store binary) as such:
```
declare "${key}"="$(echo ${creds} | jq -r .access_key)"
declare "${secret}"="$(echo ${creds} | jq -r .secret_key)"
```

Reply:
👍 sounds good